node/npm/glide/index.ts

```diff
@@ -11,64 +11,73 @@ let globalObject = global as unknown;
 
 /* eslint-disable @typescript-eslint/no-require-imports */
 function loadNativeBinding() {
-    let nativeBinding = null;
+    let nativeStr = process.env.native_binding;
 
-    switch (platform) {
-        case "linux":
-            switch (arch) {
-                case "x64":
-                    switch (familySync()) {
-                        case GLIBC:
-                            nativeBinding = require("@scope/valkey-glide-linux-x64");
-                            break;
-                        case MUSL:
-                            nativeBinding = require("@scope/valkey-glide-linux-musl-x64");
-                            break;
-                        default:
-                            nativeBinding = require("@scope/valkey-glide-linux-x64");
-                            break;
-                    }
+    if (nativeStr == undefined) {
+        switch (platform) {
+            case "linux":
+                switch (arch) {
+                    case "x64":
+                        switch (familySync()) {
+                            case MUSL:
+                                nativeStr = "linux-musl-x64";
+                                break;
+                            case GLIBC:
```

Comment:
I think you can avoid repeating this part by replacing with
```ts
const prefix = familySync() == GLIBC ? "" : "-musl";
```
furthermore, you can simplify nested switches and build the string with components:
```ts
const nativeStr = `${platform}${prefix}-${arch}`;
```
That's more reliable and it is harder to do a mistake there.